build.gradle.kts

```diff
@@ -19,12 +19,12 @@ configurations {
 dependencies {
   implementation("com.google.guava:guava:32.1.2-jre")
   implementation("commons-validator:commons-validator:1.9.0")
-  implementation("com.googlecode.libphonenumber:libphonenumber:8.13.41")
+  implementation("com.googlecode.libphonenumber:libphonenumber:8.13.45")
   implementation("org.springframework.boot:spring-boot-starter-webflux")
   implementation("org.springframework.boot:spring-boot-starter-security")
   implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
   implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
-  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:3.1.1")
+  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:4.4.2")
```

Comment:
@jnioche-jd 
I think it is recommended to update to this version.
There is alerts message for this artefact, i have commented them this PR.